src/fidasim.f90

```diff
@@ -1722,7 +1735,7 @@ subroutine read_inputs
     inquire(file=namelist_file,exist=exis)
     if(.not.exis) then
         write(*,'(a,a)') 'READ_INPUTS: Input file does not exist: ', trim(namelist_file)
-        error stop
+        stop
```

Comment:
I was under the impression that `error stop` was necessary for MPI code so that all the processes would exit. Does changing this break the MPI code?

Reply:
I had just verified that compiles.

Is this a requirement of the CoArray library?
I never had this issue with regular MPI.

Anyhow, I am open to suggestions on how to change it.
PGI fortran simply labels it as a syntax error.

Reply:
BTW: Just tested this branch with gcc+MPI, and I don't see any difference in behaviour on error.
In both cases (master and this branch) the MPI ensemble seems to hang indefinitely.
The only difference is the message in stderr.